packages/authentication/src/types.ts

```diff
@@ -22,3 +22,9 @@ export interface UserProfile {
   name?: string;
   email?: string;
 }
+
+export interface AuthenticationStrategy {
+  name: string;
+  isPassportStrategy?: boolean;
```

Comment:
-1

AFAIK, passport strategies use different API contract than LB4 `AuthenticationStrategy`. IMO, we should tell people to use `StrategyAdapter` to convert Passport strategies into LB strategies.

Reply:
@bajtos yeah that flag is added at the wrong place, see my comment https://github.com/strongloop/loopback-next/commit/53cfeebbaf2ebe2897e28f51432f6fb5d1a4e946#r33223459 .